### DISCUSSION TITLE
Onboarding new teammates / designers to a file & syncing tokens

### DISCUSSION THREAD

**Discussion by danhannigan** [six7/figma-tokens#700] [Q&A] [answered]
Hi there! I've been poking around with the plugin for a bit and we're looking to use it for syncing our Design Tokens among our team in Figma, but I'm running into a workflow that just isn't clear to me but I'm sure it's right in front of my face and that is: how do new teammates see the tokens? 

The scenario we experienced was: 
- I made a new Figma file and made some new Tokens in the plugin via JSON, got the plugin hooked up to Github, and deployed them. 
- I then had another teammate hop into the Figma file and when they opened the plugin pane it was empty (as in they had none of the tokens), I guess that didn't match my expectations, but what's the process for getting them to sync the Tokens? Would they have to have a GitHub account and a personal access token hooked up in the settings? We clicked around on Load (didn't do what we expected), and "Styles" to import or create, nothing seemed to make the connection though.

Sorry if that's a goofy/obvious question and the answer is right there — I poked around in the docs for a bit and am still scratching my head.

**Comment by danhannigan** [comment]
I think I answered my own question in a way after some more digging. We swapped to "Local Storage" and now anyone who hops into the file and has the plugin can view the token set.

**Comment by six7** [comment, marked as answer]
Your teammate would need to add an access token as well as we dont save those on the file. Instead we save everything but the access token, so security happens only on the user level.

If you do not need to use multiple files, local storage is fine. However if you'd like to use the same source of truth as dev, then github is the better choice (in addition to confidence that your tokens are version controlled).

**Reply by danhannigan** [reply]
Ah that makes sense! Thanks for taking the time to provide an answer!
